pyscitt/pyscitt/cli/client_arguments.py

```diff
@@ -88,6 +94,10 @@ def create_client(args: argparse.Namespace):
         key = args.member_key.read_text()
         kwargs["member_auth"] = (cert, key)
 
+    if "akv_configuration" in args and args.akv_configuration:
+        akv_configuration = args.akv_configuration.read_text()
+        kwargs["member_auth"] = (None, None, akv_configuration)
+
```

Comment:
As it is now, even if an AKV configuration is specified it will try to read the cert and key at the default paths, and fail if the files don't exist.

I think we should check AKV first, and only if it is not specified try to read the cert/key.